README.md

```diff
@@ -70,3 +70,32 @@ You should see output similar to the following:
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
 
+
+## Using Vagrant
+You can create a new Hypervisor installed with your app and in running mode.
+* Download and install VirtualBox on your host https://www.virtualbox.org/wiki/Downloads
+* Download and install Vagrant: https://www.vagrantup.com/downloads 
+* Open powershell 
+* Go to the root of your TODO app
+* Run the following on your command line to create the hypervisor with the TODO app installed and running:
+```bash
+vagrant up --provision
+```
+* Once complete login to the hypervisor created (username & password: vagrant/vagrant)
+```bash
+vagrant ssh
+```
+* Then tail the gunicorn access logs:
+```bash
+cd /vagrant
+tail -f ./gunicorn-access.log
```

Comment:
You can tail it from your own machine rather than the VM since you're creating the log file in the shared directory. Though the command would be different in powershell.